bindings/core/src/method/client.rs

```diff
@@ -224,25 +226,31 @@ pub enum ClientMethod {
         /// Block ID
         block_id: BlockId,
     },
-    /// Get block raw
+    /// Get block as raw bytes.
     #[serde(rename_all = "camelCase")]
     GetBlockRaw {
         /// Block ID
         block_id: BlockId,
     },
-    /// Get output
+    /// Get output with its output ID proof.
```

Comment:
Is this a TODO for when the PR is merged?

Reply:
well, that's the request side, right? We're just providing the `output_id` here, and the response will hold the output and proof.

Reply:
omg lol I am so dumb